pkg/webhook/webhook.go

```diff
@@ -157,49 +164,80 @@ func (w *Config) Start(ctx context.Context) error {
 
 // createFleetWebhookConfiguration creates the ValidatingWebhookConfiguration object for the webhook.
 func (w *Config) createFleetWebhookConfiguration(ctx context.Context) error {
-	whCfg := admv1.ValidatingWebhookConfiguration{
+	vWhCfg := admv1.ValidatingWebhookConfiguration{
```

Comment:
```suggestion
	validatingWebhookConfig := admv1.ValidatingWebhookConfiguration{
```

prefer to use a meaningful name. Please do not shorten the variable names by picking letters.